eng/pipelines/templates/jobs/aggregate-reports.yml

```diff
@@ -11,6 +11,10 @@ resources:
 trigger: none
```

Comment:
Orthogonal but generally pipeline entry-points shouldn't be under the templates folder as they aren't a reusable template. I've mostly been putting pipelines at in `eng/pipelines` folder directly.

Reply:
Yeah this is a good suggestion. I'll move it with this PR. There are additional changes I need to make as the conditional injection I'm using simply doesn't work if it's not isolated to a single job. :/

Reply:
@weshaggard I tested the job prior to the commit moving it. [Works as intended](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=414186&view=logs&j=0874ffeb-2919-5b4b-20de-16a97970b94c)

MY adding of commits cleared your approval. When you see this comment can you approve this again?

Reply:
I'll update the actual build def to point at the new location after I merge it to master.